kf_lib_data_ingest/etl/load/load.py

```diff
@@ -348,11 +348,13 @@ def _run(self, target_entities):
                 'DRY RUN mode is ON. No entities will be loaded into the '
                 'target service.'
             )
-        for entity_type, entities in target_entities.items():
+        for entity_type in self.target_api_config.target_concepts.keys():
             if entity_type not in self.entities_to_load:
                 self.logger.info(f'Skipping load of {entity_type}')
                 continue
 
+            entities = target_entities[entity_type]
```

Comment:
and if entity_type isn't in target_entities?

Reply:
👍

Reply:
this does not appear to be resolved

Reply:
oops tests are failing bc some of the test study data has to be changed. some ingest package configs have study_file and family_relationship in the list of entities to load